docs/client.md

```diff
@@ -235,7 +235,7 @@ This does not estimate the gas cost of updating the allowance of an ERC20 token<
     * [.hasPermission(params)](#ClientMethods+hasPermission) ⇒ <code>\*</code>
     * [.getDao(daoAddressOrEns)](#ClientMethods+getDao) ⇒ <code>\*</code>
     * [.getDaos({)](#ClientMethods+getDaos) ⇒ <code>\*</code>
-    * [.getDaoBalances(daoAddressorEns)](#ClientMethods+getDaoBalances) ⇒ <code>\*</code>
+    * [.getDaoBalances({)](#ClientMethods+getDaoBalances) ⇒ <code>\*</code>
```

Comment:
Don't we have this parameter now??

Reply:
Yes bbut it seems that this is a problem with the jsdoc2md because I noticed it happens in other places too

Reply:
but is this parameter actually gone now?  I think it isn't

If this is a problem with the tool, we close this comment and look into it later

Reply:
Yes instead of `{` It should appear `DaoBalancesQueryParams`. I think It has to do with show we document the default values